data/secscan_model/secscan_v4_model.py

```diff
@@ -196,9 +204,8 @@ def needs_reindexing_query(indexer_hash):
             )
 
         # 4^log10(total) gives us a scalable batch size into the billions.
-        batch_size = self.app.config.get(
-            "SECURITY_SCANNER_V4_BATCH_SIZE", int(4 ** log10(max(10, max_id - min_id)))
```

Comment:
Are you removing config param batch size from here? Where are supplying that from?

Reply:
It's being passed as a param, since the caller is already looking at `SECURITY_SCANNER_V4_BATCH_SIZE`.

Reply:
good change, as this was confusing when reading this the other day.